src/main/scala/org/broadinstitute/dsde/firecloud/utils/StreamingPassthrough.scala

```diff
@@ -135,9 +148,8 @@ trait StreamingPassthrough
 
     // find every part of the actual request path, minus the base.
     val baseString = localBasePath.toString
-    val requestString = requestUri.path.toString
+    val requestString = if(modifiedRemotePath.isEmpty) requestUri.path.toString() else modifiedRemotePath.get
```

Comment:
We generally want to avoid direct `get` calls, even in cases like this where logically it's impossible for them to throw an error. The Scala way to do this is:
```
val requestString = modifiedRemotePath.getOrElse(requestUri.path.toString)
```